stacker/context.py

```diff
@@ -50,6 +54,8 @@ def get_stacks(self):
                     context=self,
                     parameters=self.parameters,
                     mappings=self.mappings,
+                    force=stack_def['name'] in self.force_stacks,
+                    locked=stack_def.get('locked', False),
```

Comment:
There could be some confusion here if someone puts `locked: FALse` or something similar, since that will get evaluated to the string `FALse` rather than the python False, and so it will be locked.  That said, I think a worse case would be someone including `TRue` and not getting True.  Not sure the best way to handle this.